tools/driver/CMakeLists.txt

```diff
@@ -51,6 +51,10 @@ endif()
 
 add_dependencies(clang clang-headers)
 
+if (DEFINED CHECKEDC_IN_TREE)
+  add_dependencies(checkedc-headers)
```

Comment:
This should be `add_dependencies(clang checkedc-headers)` right? I think with just one argument isn't valid.